src/sections/header.liquid

```diff
@@ -79,13 +79,13 @@
     {% endif %}
 
     <form action="/search" method="get" role="search">
-      <label for="Search" class="label-hidden">
+      <label for="Search-{{ section.id }}" class="label-hidden">
```

Comment:
This ID will always be `header`. Does it make sense to add the ID here? What's your reasoning over adding it to the other instance within the page content?

Reply:
to me, `id="Search"` make more sense on `search.liquid` but it's pretty arbitrary at that point.

Made sense to keep the `SectionXXX-{{ section.id }}` pattern when `section.id` is available.

Unless i'm mistaken, there's no `{{ section.id }}` for `search.liquid` ?

Reply:
If it's a static section, `{{ section.id }}` will return the name of the section. So because `header.liquid` is a static section in this case, it'll become `Search-header`.

Reply:
👍 Sounds good to me.